app/controllers/carts_controller.rb

```diff
@@ -21,6 +21,13 @@ def show
     @courses = @cart.find_by_course_id
   end
 
+  def add
```

Comment:
This works for now but let's make sure it gets refactored to apply RESTful conventions.